graph-proxy/src/graphql/workflows.rs

```diff
@@ -131,13 +135,14 @@ impl WorkflowRunningStatus {
     fn new(
         value: IoArgoprojWorkflowV1alpha1WorkflowStatus,
         metadata: Arc<Metadata>,
+        server_url: Url,
```

Comment:
I don't love passing the `server_url` in here, we should retrieve it from the `Context` of the `url` resolver